### ISSUE TITLE
Frontend testing

### ISSUE THREAD

**Issue by easbar** [graphhopper/graphhopper-maps#96]
This could be useful for things like "does the camera keep the position when we change the map layer etc.". But I'm also a bit skeptical if the benefits of this will outweigh the costs. Maybe at some point we could check what it would take to add a few tests of this kind.

**Comment by Janekdererste** [comment]
yes, I think we should have something like this. I didn't add test for components yet, since everything was in motion all the time. But I guess we could start with components where we think they won't change anymore, or not so often. 

A first candidate could be the address input component. It is quite complex but I think the functionality won't change so much in the futre.
